PhoneAuthUI/Sources/User/UserConsentScreen.swift

```diff
@@ -0,0 +1,156 @@
+//
```

Comment:
Was hat der ConsentScreen mit `PhoneAuthUI` zutun? Ist es nicht im falschen Package?

Reply:
Wird in ein User Package umgezogen.